ludwig/features/binary_feature.py

```diff
@@ -231,7 +237,7 @@ def get_prediction_set(self):
 
     @classmethod
     def get_output_dtype(cls):
-        return tf.bool
+        return torch.bool
```

Comment:
Could you also add an `input_shape` property for this feature?

Reply:
added